Jenkinsfile

```diff
@@ -342,6 +249,50 @@ ccache -s
             } //builds item
         } //for variants
 
+        // Also add a single build job for doing the RPM build
+        // on a docker node
+        builds['rpm'] = {
+            node('docker') {
+                def bldlabel = 'rpm'
+                configFileProvider (
+                    [configFile(
+                        fileId: 'rippled-commit-signer-public-keys.txt',
+                        variable: 'SIGNER_PUBLIC_KEYS')])
```

Comment:
Thinking out loud:
We can exclude this file and allow an rpm to be built for commits from external developers
https://github.com/ripple/rippled-package-builder/blob/5578d26221e294109fdf79e0d75f90ed0bdb9f2e/rpm-builder/build_rpm.sh#L34
However, I think I prefer using the file so we don't get surprised trying to build an rpm after merging a commit missing a signature from a whitelisted key.